README.md

```diff
@@ -73,7 +71,8 @@ Make sure your client provides a implementation of [`fetch`](https://developer.m
 - [`*auth.unlink(authProvider)`](https://redux-saga-firebase.js.org/reference/auth#unlink)
 - [`*auth.updateEmail(email)`](https://redux-saga-firebase.js.org/reference/auth#updateEmail)
 - [`*auth.updatePassword(password)`](https://redux-saga-firebase.js.org/reference/auth#updatePassword)
-- [`*auth.updateProfile(profile)`](https://redux-saga-firebase.js.org/reference/auth#updateProfile)
+- [`*auth.updateProfile(profile)`](https://redux-saga-firebase.js.org/reference/auth#deleteProfile)
+- [`*auth.deleteProfile()`](https://redux-saga-firebase.js.org/reference/auth#deleteProfile)
```

Comment:
can you keep the methods ordered alphabetically please?

Reply:
of course